somnio_starter/__brick__/{{project_name.snakeCase()}}/packages/user_repository/test/src/models/user_test.dart

```diff
@@ -3,9 +3,12 @@ import 'package:user_repository/user_repository.dart';
 
 void main() {
   group('User', () {
+    const validId = '123';
+    const validEmail = 'test@example.com';
+
     test('constructor initializes properties', () {
-      const id = '123';
-      const email = 'test@example.com';
+      const id = validId;
+      const email = validEmail;
```

Comment:
I think you can avoid this since the values are already defined as global variables